### PR TITLE
6-Rename-RuRootRule-as-RuRulesManager

### LINKED ISSUE #6: Rename RuRootRule as RuRulesManager
I think it would be better for understanding of how the project works. :-)

In the doc, you name the local var holding the root rule "rulesManager" anyway.

### PR DESCRIPTION
**Pull request by juliendelplanque**
Fixes #6 

Renamed RuRootRule  as RuRulesManager.
Renamed test class accordingly.
Fixed occurences in comments.

### CODE REVIEW

**Comment by jecisc** [comment]
Documentation should also be updated.

**Comment by juliendelplanque** [comment]
Done @jecisc

File: README.md [hunk 1]
```diff
@@ -34,14 +34,14 @@ Note you can replace the #master by another branch such as #development or a tag
 
 ## Quick start
 
-To create a batch of rules to run on a model you first need to create a `RuRootRule`.
+To create a batch of rules to run on a model you first need to create a `RuRulesManager`.
 
 A root rule is a composite rule that will know the model which the rules should apply and that contains all the rules. It can be considered as the rules manager.
```
Comment (jecisc) [inline README.md]:
```suggestion
A rule manager is a composite rule that will know the model which the rules should apply and that contains all the rules.
```

File: documentation/Documentation.md [hunk 2]
```diff
@@ -18,14 +18,14 @@ This project provides objects to represent rulesto not violate on a model and al
 
 ## Quick start
 
-To create a batch of rules to run on a model you first need to create a `RuRootRule`.
+To create a batch of rules to run on a model you first need to create a `RuRulesManager`.
 
 A root rule is a composite rule that will know the model which the rules should apply and that contains all the rules. It can be considered as the rules manager.
```
Comment (jecisc) [inline documentation/Documentation.md]:
```suggestion
A rule manager is a composite rule that will know the model which the rules should apply and that contains all the rules.
```

**Review by jecisc — CHANGES_REQUESTED** [review]

**Comment by juliendelplanque** [comment]
I just did a stupid find and replace :D. Sorry for the ones I missed.

**Review by jecisc — APPROVED** [review]

**Pull request merged**